pkg/networkmanager/interfaces.js

```diff
@@ -619,6 +619,26 @@ function NetworkManagerModel() {
                           };
         }
 
+        function JSON_parse_carefully(str) {
+            try {
+                return JSON.parse(str);
+            }
+            catch (e) {
+                return { };
```

Comment:
In what cases does this happen? Does it happen as a result of an admin editing config files manually? If so, we should probably display an message instead of an empty config.